src/snowflake/snowpark/mock/mock_analyzer.py

```diff
@@ -141,7 +141,6 @@
     MockSelectable,
     MockSelectableEntity,
     MockSelectExecutionPlan,
-    MockSelectSnowflakePlan,
```

Comment:
based upon our discussion, this class can be removed from `mock_select_statement.py`?

Reply:
Yes, I could include this in this PR.